lib/solargraph/pin/namespace.rb

```diff
@@ -89,6 +91,11 @@ def gates
           [path] + @open_gates
         end
       end
+
+      # @return [Array<String>]
+      def parameters
```

Comment:
@castwide would it make sense to rename this to something like:
```suggestion
      def generics
```

or

```suggestion
      def generic_parameters
```

The current `def parameters` interface seems to be incompatible with other pins anyway with the return types:

https://github.com/castwide/solargraph/blob/99801d1aaea62c28a6ac6c558c812c50c1e81fe5/lib/solargraph/pin/block.rb#L29-L32